torch_geometric/utils/homophily.py

```diff
@@ -37,17 +37,30 @@ def homophily(edge_index: Adj, y: Tensor, batch: OptTensor = None,
 
       That measure is called the *node homophily ratio*.
 
+    - In the "Large-scale learning on non-homophilous graphs: \
+      New benchmarks and strong simple methods" paper, the class insensitive 
+      homophily metric better captures the presence or absence of homophily:
+
+      .. math::
+        \text{homophily} = \frac{1}{C-1}\sum_{k=0}^{C-1}\begin{bmatrix}
+        {h_k - \frac{\lvert C_k \rvert}{n}}\end{bmatrix}_+,
+        
+      .. math::
+        h_k = \frac{\sum_{u \in C_k}d_u^{(k_u)}}{\sum_{u \in C_k}d_u}
```

Comment:
Can we add some more information to this formula? For example about `C_k` or `d_u`

Reply:
We also build doc as part of our CI, see https://pytorch-geometric--3977.org.readthedocs.build/en/3977/